docs/datatypes/io.md

```diff
@@ -161,3 +161,41 @@ val action: Task[String] = Task.effectTotal(i += 1) *> Task.fail(new Throwable("
 val cleanupAction: UIO[Unit] = UIO.effectTotal(i -= 1)
 val composite = action.ensuring(cleanupAction)
 ```
+### A full working example on using brackets
+```scala mdoc:silent
+
+import scalaz.zio.{ App, DefaultRuntime, Task, UIO }
+
+import java.io.{ File, FileInputStream }
+import java.nio.charset.StandardCharsets
+
+object Main extends App {
+
+  val rt = new DefaultRuntime {}
+
+  // run my bracket
+  def run(args: List[String]) =
+    mybracket.orDie.map(_ => 0)
+
+  def closeStream(is: FileInputStream) =
+    UIO.effectTotal(is.close())
+
+  def convertBytes(is: FileInputStream) =
+    Task.effect(println(new String(is.readAllBytes(), StandardCharsets.UTF_8)))
+
+  val mybracket: UIO[Unit] = {
+
+    UIO {
+
+      val file = new File("/tmp/hello")
+
+      val string = Task.effect(new FileInputStream(file)).bracket(closeStream)(convertBytes).orDie
+
+      // Print effect
+      rt.unsafeRun(string)
```

Comment:
Please refactor the example to not use `rt.unsafeRun`, as that is not something we encourage.

In fact, since the example is using `App`, you don't need to instantiate a runtime at all.